app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -227,6 +227,21 @@ struct DeviceDelegateOpenXR::State {
     viewConfig.resize(viewCount, {XR_TYPE_VIEW_CONFIGURATION_VIEW});
     CHECK_XRCMD(xrEnumerateViewConfigurationViews(instance, system, viewConfigType, viewCount, &viewCount, viewConfig.data()));
 
+#ifdef PICOOXR
+    // The Pico 4 is much more capable than it advertises via OpenXR, and the primary device used
+    // with Pico OXR builds. We thus bump resulutions by 1.4 for a drastic improvement in image clarity.
```

Comment:
This is interesting, however it should only apply to Pico4 devices. Note that this new flavour should be usable also for devices with different resolutions.

Reply:
It should work on all devices, but of course waste performance on devices that do not benefit from the higher resolution.

If you know of a certain way to identify the Pico 4, feel free to suggest something? The main reason I specified a factor instead of a hardcoded resolution was that I hoped that the Neo 3 recommends a smaller resolution. (The other was that I think it would be nice to make that factor a setting for users in the long term, on all headsets.)

Reply:
This honestly seems like a runtime bug that shouldn't be worked around in applications

Reply:
@rpavlik is Pico using Monado? If that's the case I have another one to report :smile:

Reply:
Not unmodified at least.